workbench-core/base/src/aws/helpers/s3Service.ts

```diff
@@ -75,4 +76,13 @@ export default class S3Service {
 
     await recursiveUpload(path, '');
   }
+
+  public async getPresignedUrl(s3BucketName: string, key: string, expirationMinutes: number) : Promise<string> {
+    // Sign the url
+    const command = new GetObjectCommand( {
+      Bucket: s3BucketName,
+      Key:key,
+    });
+    return getSignedUrl(this._s3, command, { expiresIn: expirationMinutes });
```

Comment:
the `expiresIn` field is measured in seconds, not minutes: https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/interfaces/_aws_sdk_types.requestpresigningarguments-1.html